css/style.css

```diff
@@ -1,19 +1,189 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
+/* --==================== general styles ====================-- */
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+h1, h2, .about-text{
+  font-weight: lighter;
+}
+
+a, u{
+  color: white;
+}
+ /* --==================== HEADER ====================-- */
+ .header_logo {
+    height: 40px;
+    float: left;
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced more recently). 
Set `display: flex` on the `header` instead